src/cdk/v2/destinations/bluecore/data/bluecoreTrackConfig.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+    "destKey": "properties.distinct_id",
+    "sourceKeys": ["userId", "anonymousId", "email"],
```

Comment:
Also shouldn't the order of id be same for identify and track config?

Reply:
removed from here